src/components/website-management/website-form-components/customization/customization-components/CookieBanner.js

```diff
@@ -0,0 +1,331 @@
+import React, { useContext, useEffect, useState } from "react";
+import Typography from "@mui/material/Typography";
+import { Box } from "@mui/system";
+import { Button } from "@mui/material";
+import { BannerContext } from "./BannerContext";
+import CustSelectBoxBanner from "./SelectBoxBanner";
+import axios from "axios";
+import Snackbar from "@mui/material/Snackbar";
+import MuiAlert from "@mui/material/Alert";
+import CookieSettingsDialog from "./CookieSettingDialog";
+import BannerContentSelectBox from "./BannerContentSelectBox";
+
+const CookieBannner = () => {
+
+  const passedId = useContext(BannerContext).websiteId;
+  const { setColor, setTop, setLeft, setChosenCont, color, textColor, setTextColor} = useContext(BannerContext);
+
+  const [templates, settemplates] = useState([]);
+  const [banner, setBanner] = useState([]);
+  const [selectedTemplate, setSelectedTemplate] = useState({});
+
+  const [alignmentValue, setAlignmentValue] = useState("");
+  const [positionValue, setPositionValue] = useState("");
+  const [colorValue, setColorValue] = useState("");
+  const [textColorValue, setTextColorValue] = useState("");
+
+  /*Sanck bar useStates*/
+  const [snackbarOpen, setSnackbarOpen] = useState(false);
+  const [snackbarMessage, setSnackbarMessage] = useState("");
+  const [error, setError] = useState(false);
+  const [errorMessage, setErrorMessage] = useState("");
+  const [showDialog, setShowDialog] = useState(false);
+
+  const handleCloseClick = () => {
+    setShowDialog(false);
+  };
+
+  const handleCustomizeClick = () => {
+    setShowDialog((prevShowDialog) => !prevShowDialog);
+  };
+
+  const loadTemplates = async () => {
+    try {
+      const websiteID = passedId;
+      const result = await axios.get(
+          `http://localhost:8080/api/v1/templates/${websiteID}`
+      );
+      settemplates(result.data);
+
+      if (Object.keys(selectedTemplate).length !== 0) {
+        const template = result.data.find(
+            (t) => t.templateName === selectedTemplate?.templateName
+        );
+        setSelectedTemplate(template);
+      } else {
+        const template = result.data[0];
+        setSelectedTemplate(template);
+      }
+    }
+    catch (error) {
+      console.error("Error loading banners: ", error);
+    }
+  };
+
+  const laodBanner = async () => {
+
+    try {
+      const websiteId = passedId;
+      const response = await axios.get(
+        `http://localhost:8080/api/v1/banners/${websiteId}`
```

Comment:
Let's define `http://localhost:8080` as the base URL at the app entry point. Then we wouldn't need to specify it every time when calling backend.

Ref: https://axios-http.com/docs/config_defaults

Reply:
noted